### PR TITLE
graduate LegacyServiceAccountTokenTracking to GA

### PR DESCRIPTION
**Pull request by zshihang**
#### What type of PR is this?

/kind feature

#### Does this PR introduce a user-facing change?
```release-note
Graduated the `LegacyServiceAccountTokenTracking` feature gate to GA. The usage of auto-generated secret-based service account token now produces warnings, and relevant Secrets are labeled with a last-used timestamp (label key `kubernetes.io/legacy-token-last-used`).
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token
```

### CODE REVIEW

**Comment by zshihang** [comment]
cc @liggitt for review

**Comment by zshihang** [comment]
cc @liggitt

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	LegacyServiceAccountTokenNoAutoGeneration: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.29
 
-	LegacyServiceAccountTokenTracking: {Default: true, PreRelease: featuregate.Beta},
+	LegacyServiceAccountTokenTracking: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.29
```
Comment (liggitt) [inline pkg/features/kube_features.go:1036]: two releases, so 1.30

```suggestion
	LegacyServiceAccountTokenTracking: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.30
```

**Review by liggitt — COMMENTED** [review]
thanks, one nit on the feature gate lock, lgtm otherwise

before merge, open an update to the KEP checking off the beta → GA criteria (https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token#beta---ga-graduation-1)

* sweep for any bug reports
* verify multiple providers are deploying 1.27 with the tracking enabled (GKE is, check with contacts at other providers)
* ping PRR to ack the GA graduation of this aspect of the KEP

**Pull request merged**